src/visualization/types/Band/options.tsx

```diff
@@ -33,15 +32,15 @@ import {BandViewProperties} from 'src/types'
 import {VisualizationOptionProps} from 'src/visualization'
 
 const {BASE_2, BASE_10} = AXES_SCALE_OPTIONS
-const UPPER_COLUMN_PLACEHOLDER = `Please enter upper column's function name`
-const MAIN_COLUMN_PLACEHOLDER = `Please enter main column's function name`
-const LOWER_COLUMN_PLACEHOLDER = `Please enter lower column's function name`
+const REMOVE_COLUMN = '(none)'
 
 interface Props extends VisualizationOptionProps {
   properties: BandViewProperties
 }
 
-const BandViewOptions: FC<Props> = ({properties, results, update}) => {
+const BandViewOptions: FC<Props> = props => {
```

Comment:
curious about the change from destructuring the props

Reply:
Oops, that's my bad. It was not an intended change. I had a `console.log` for the whole `props`. I removed the `console.log` but didn't restore the destructuring.

I will put it back.